ci-operator/step-registry/servicemesh/istio/post/prod/servicemesh-istio-post-prod-commands.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
These should be performed by the test

Reply:
Well, that's a temporary workaround for operators installation from production. 

maistra-test-tool may test pre-release builds (operators from IIB) but we can't leak internal registry source on GH. 
That's why I didn't include any operator installation step in the past and mentioned that operators installation was a Prerequisite before running maistra-test-tool.

Marko may have some idea later and then update the maistra-test-tool for operators step.

Reply:
I asked a question about this topic in slack. If they are ok to add a make target, I will remove this "prod-commands" step in this PR later.

Reply:
What I am trying to say is that we don't need those commands here. Whataver is necessary to to run the tests should be committed in the repo itself, not here. If that's not clear, let's talk on google meet/slack. Ping me.